src/lava/magma/compiler/compiler.py

```diff
@@ -16,11 +16,11 @@
 
 import lava.magma.compiler.exceptions as ex
 import lava.magma.compiler.exec_var as exec_var
-from lava.magma.compiler.builders.builder import ChannelBuilderMp
-from lava.magma.compiler.builders.builder import PyProcessBuilder, \
+from lava.magma.compiler.builder.builder import ChannelBuilderMp
+from lava.magma.compiler.builder.builder import PyProcessBuilder, \
     AbstractRuntimeServiceBuilder, RuntimeServiceBuilder, \
     AbstractChannelBuilder, ServiceChannelBuilderMp
-from lava.magma.compiler.builders.builder import RuntimeChannelBuilderMp
+from lava.magma.compiler.builder.builder import RuntimeChannelBuilderMp
```

Comment:
Wouldn't "builders" be more appropriate because this package contains multiple builders?

Reply:
I used builders originally. I'll restore to that version.

I was trying to match some other module naming in lava, it seems we are inconsistent with naming.